examples/custom_transform_and_model.ipynb

```diff
@@ -202,6 +202,16 @@
     "ts.head(5)"
```

Comment:
<p>Don't we show both implemented transforms and that results for them are the same?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Done